index.html

```diff
@@ -31,7 +31,7 @@
 	<aside id="sticky-generator">
 	<a name="jump"></a> 
 			<h1>Sticky Generator</h1>
-			<div class="sticky-template">
+			<div class="sticky" id="sticky-template">
```

Comment:
I changed `sticky-template` to an id because the word template makes me think there should only be one of template element. All the other stickies are based on that one template. Using the class selector means that that there can be multiple elements that have the same class, but no two elements can have the same id.